lib/compute/index.js

```diff
@@ -306,6 +312,70 @@ Compute.prototype.createNetwork = function(name, config, callback) {
 };
 
 /**
+ * Create a backend service.
+ *
+ * @resource [Backend Services Overview]{@link https://cloud.google.com/compute/docs/load-balancing/http/backend-service}
+ * @resource [BackendServices: insert API Documentation]{@link https://cloud.google.com/compute/docs/reference/v1/backendServices/insert}
+ *
+ * @param {string} name - Name of the backend service.
+ * @param {object} config - See a
+ *     [BackendService resource](https://cloud.google.com/compute/docs/reference/v1/backendServices#resource).
+ * @param {function=} callback - The callback function.
+ * @param {?error} callback.err - An error returned while making this request.
+ * @param {module:compute/network} callback.network - The created Network
```

Comment:
Is this correct? I'm not familiar with the upstream counterpart, but if it does create a network I would think this method should be titled `createNetwork`.